jwst/tweakreg/tweakreg_step.py

```diff
@@ -56,21 +56,35 @@ class TweakRegStep(Step):
         use_custom_catalogs = boolean(default=False) # Use custom user-provided catalogs?
         catalog_format = string(default='ecsv') # Catalog output file format
         catfile = string(default='') # Name of the file with a list of custom user-provided catalogs
+        starfinder = option('dao', 'iraf', 'segmentation', default='dao') # Star finder to use.
+        snr_threshold = float(default=10.0) # SNR threshold above the bkg for star finder
+        # kwargs for DAOStarFinder and IRAFStarFinder, only used if starfinder is 'dao' or 'iraf'
         kernel_fwhm = float(default=2.5) # Gaussian kernel FWHM in pixels
-        snr_threshold = float(default=10.0) # SNR threshold above the bkg
+        minsep_fwhm = float(default=0.0) # Minimum separation between detected objects in FWHM
```

Comment:
Is a value of zero a suitable default, as opposed to something on the order of 1.0 or more?

Reply:
In the tests I did, it doesn't matter much one way or another.  The reason I left it as-is was because the tests in test_nircam_image.py and test_miri_image.py expected 0.0 to be the default.  It's worth noting that when this is set to zero, DAOStarFinder or IRAFStarFinder will actually use a minimum value of 2 pixels.  But if you feel strongly, I'm happy to change it to 1.0 and explicitly pass the parameter in all the relevant tests.

Reply:
No change necessary. Was just curious as to whether that was a reasonable choice. Given your statement that two of the methods actually set the min to 2 pix, sounds reasonable.